internal/serve/graphql/schema/statechange.graphqls

```diff
@@ -0,0 +1,38 @@
+# GraphQL StateChange type - represents changes to blockchain state
+# This type has many nullable fields to handle various state change scenarios
+# TODO: Break state change type into interface design and add sub types that implement the interface
+type StateChange{
+  id:                         String!
+  accountId:                  String!           
+  stateChangeCategory:        StateChangeCategory!
+  stateChangeReason:          StateChangeReason
+  ingestedAt:                 Time!
+  ledgerCreatedAt:            Time!
+  ledgerNumber:               UInt32!           
```

Comment:
Why is `ledgerNumber` available on state changes but not operations or transactions?

IMO we need to have all the meta fields (`ingestedAt`, `ledgerCreatedAt`, and `ledgerNumber`) on all 3 structs or only on transactions.